lib/context.js

```diff
@@ -350,6 +350,11 @@ api.createTermDefinition = ({
       'Invalid JSON-LD syntax; keywords cannot be overridden.',
       'jsonld.SyntaxError',
       {code: 'keyword redefinition', context: localCtx, term});
+  } else if(term.match(/@[a-zA-Z]+$/)) {
+    // FIXME: remove logging and use a handler
+    console.warn('WARNING: terms beginning with "@" are reserved' +
```

Comment:
Using `console.warn` here is the same issue that resulted in the `protectedMode` `warn/error` flag.  That was supposed to eventually change a better handler, but that hasn't been designed yet.  Might want another similar flag and we can fix them all at once.  The machinery to pass around those options still exists so would be easy to add.  I'm not sure what the best behavior even is.

Reply:
I'll leave this for you to come up with a comprehensive solution. There are other areas where the spec says to generate warnings.

Reply:
Ok.  Printing out warnings to a console can be invisible for backend code and UIs, hence the idea to have some of handler.  I haven't put time into a design for that.  A more strict mode that throws errors seems useful to me as well.